Wurst Client for MC 1.9.X/src/tk/wurst_client/mods/PanicMod.java

```diff
@@ -25,17 +26,27 @@ public void onEnable()
 		wurst.events.add(UpdateListener.class, this);
 	}
 	
+	List<Mod> panicedMods = new ArrayList<>();

```

Comment:
I kindly notice you that the compiler will not compile on this line. You fix it on your own!